### PR TITLE
Updated Email Contents

### PR DESCRIPTION
**Pull request by deaspo**
I have updated the email contents for the following user activation, vm status changed, password resset email, newly booked vm, vm charged and utils.views (site-name)

### CODE REVIEW

**Comment by verysanghee** [comment]
@deaspo in .po DE parts look to be empty.

**Comment by verysanghee** [comment]
vm_status_changed.html looks to be same texts as before, what is updated here?

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -204,36 +205,118 @@ msgstr "CHF/Monat"
 msgid "Start VM"
 msgstr "VM jetzt starten"
 
+#: templates/hosting/emails/new_booked_vm.html:99
+#: templates/hosting/emails/vm_charged.html:99
+#, python-format
+msgid " Your New VM %(vm.name)s at Data Center Light "
+msgstr "Ihre neue VM%(vm.name)s bei Data Center Light "
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: @deaspo 208-673 this part is added by you? It looks wrong. (It's in "Ihre" form, which we don't use in dcl.) May I ask where you got the text?

Comment (deaspo) [inline hosting/locale/de/LC_MESSAGES/django.po]: @verysanghee I got these texts from Google

**Pull request closed without merging**